db/migrate/20230617100117_add_call_for_evidence_response_schemas.rb

```diff
@@ -0,0 +1,34 @@
+class AddCallForEvidenceResponseSchemas < ActiveRecord::Migration[7.0]
+  def up
+    create_table "call_for_evidence_response_form_data", id: :integer, charset: "utf8mb3", force: :cascade do |t|
+      t.string "carrierwave_file"
+      t.datetime "created_at", precision: nil
+      t.datetime "updated_at", precision: nil
+    end
+
+    create_table "call_for_evidence_response_forms", id: :integer, charset: "utf8mb3", force: :cascade do |t|
+      t.string "title"
+      t.datetime "created_at", precision: nil
+      t.datetime "updated_at", precision: nil
+      t.integer "call_for_evidence_response_form_data_id", foreign_key: true
+    end
+
+    create_table "call_for_evidence_participations", id: :integer, charset: "utf8mb3", force: :cascade do |t|
+      t.integer "edition_id", foreign_key: true
+      t.string "link_url"
+      t.datetime "created_at", precision: nil
+      t.datetime "updated_at", precision: nil
+      t.string "email"
+      t.integer "call_for_evidence_response_form_id"
+      t.text "postal_address"
+      t.index %w[call_for_evidence_response_form_id], name: "index_cons_participations_on_cons_response_form_id"
```

Comment:
I don't think this index name is right? "cons" should maybe be "cfes"? What's a bit odd is I think if there's an index with the same name on the consultation participation table the migration should fail, but tests suggest that isn't happening 🤔